samples/sampler/stories/qa/Spinner.js

```diff
@@ -0,0 +1,55 @@
+import kind from '@enact/core/kind';
+import {boolean, select, text} from '@enact/storybook-utils/addons/knobs';
+import ri from '@enact/ui/resolution';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import Spinner from '@enact/agate/Spinner';
+import Skinnable from '@enact/agate/Skinnable';
+
+Spinner.displayName = 'Spinner';
+
+// Set up some defaults for info and knobs
+const
+	prop = {
+		longText:'SpinnerWithLongText SpinnerWithLongText SpinnerWithLongText'
+	};
+
+const SkinnedSpinnerBase = kind({
+	name: 'SkinSpinner',
+
+	propTypes: {
+		skinVariants: PropTypes.object
+	},
+
+	render: ({skinVariants, ...rest}) => {
+		return <Spinner {...rest} color={skinVariants.night ? 'light' : 'dark'} />;
+	}
+});
+
+const SkinnedSpinner = Skinnable({variantsProp: 'skinVariants'}, SkinnedSpinnerBase);
+
+storiesOf('Spinner', module)
+	.add(
+		'with long content',
+		() => (
+			<div>
+				<div
+					style={{
+						height: ri.scaleToRem(300),
+						border: ri.scaleToRem(6) + ' dotted blue'
+					}}
+				>
+					<SkinnedSpinner
+						transparent={boolean('transparent', Spinner, false)}
```

Comment:
The order of knob props is different from the default sampler which has alphabetically ordered props. The order could be not an alphabetical order if intended, but it looks better to have the same order between the default and QA samplers.